spec/spec_helper.rb

```diff
@@ -32,7 +32,23 @@ def stub_govuk_repos(repo_names)
     )
 end
 
-def stub_github_repo(repo_name, feature_branches: [], contents: [])
+def stub_github_get_pull_requests(repo_name, branch_name)
+  stub_request(:get, "https://api.github.com/repos/alphagov/#{repo_name}/pulls?head=alphagov:#{branch_name}&per_page=100")
+    .to_return(
+      status: 200,
+      headers: { "Content-Type": "application/json" },
+      body: [{}].to_json,
+    )
+end
+
+def stub_github_repo(repo_name, feature_branches: [], pull_request_branches: [], contents: [])
+  stub_request(:get, "https://api.github.com/repos/alphagov/#{repo_name}/pulls?head=alphagov:branch&per_page=100")
```

Comment:
It's a bit icky that we have a hardcoded "branch" name here - perhaps you can do something like:

```suggestion
def stub_github_repo(repo_name, branch_name: "branch", feature_branches: [], pull_request_branches: [], contents: [])
  stub_request(:get, "https://api.github.com/repos/alphagov/#{repo_name}/pulls?head=alphagov:#{branch_name}&per_page=100")
```

(Giving the parameter a default value of "branch" means you won't need to go through and update all the tests 👍 )